packages/frontend/app/(dashboard)/course/[cid]/(settings)/settings/chatbot_document_chunks/page.tsx

```diff
@@ -52,8 +52,15 @@ export default function ChatbotDocuments(
     }
     await API.chatbot.staffOnly
       .addDocumentChunk(courseId, body)
-      .then(() => {
-        message.success('Document added successfully.')
+      .then((addedDocs) => {
+        message.success(
+          `Document${addedDocs.length > 1 ? 's' : ''} added successfully.`,
+        )
+        if (addedDocs.length > 1) {
+          message.warning(
```

Comment:
```suggestion
          message.info(
```

Warning also works, just personal preference i guess